packages/stateless/components/proposal/ProgressBar.tsx

```diff
@@ -37,24 +38,39 @@ export const ProgressBar = ({
             height: thickness,
           }}
         >
-          {data.map(({ value, color }, index) => (
-            <div
-              key={index}
-              className="h-full"
-              style={{ width: `${value}%`, backgroundColor: color }}
-            ></div>
-          ))}
+          {data.map(({ value, color, tooltipTitle }, index) =>
+            tooltipTitle ? (
+              <Tooltip title={tooltipTitle}>
```

Comment:
the `Tooltip` component automatically hides itself when `title` is undefined/falsy! it would improve readability IMO to just wrap the div in a Tooltip like this first case here.